schedule.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+COUNT=0
+while [ true ]; do
```

Comment:
Don't use while in a script.

Instead you can set the Cron timeout appropriately to achieve the same effect. You don't really need a script. You only need to add the cron job line for this:

```sh
86400 * * * * cd /path/ && python chitragupta.py
```

cc @zorroblue